src/utils.js

```diff
@@ -20,7 +20,14 @@ const isXError = x => (error /*: Error */) => error.keyword === x;
 export const isRequiredError = isXError('required');
 export const isAnyOfError = isXError('anyOf');
 export const isEnumError = isXError('enum');
-export const getErrors = (node /*: Node*/) => (node && node.errors) || [];
+export const getErrors = (node /*: Node*/) =>
+  node && node.errors
+    ? node.errors.map(e =>
+        e.keyword === 'errorMessage'
+          ? { ...e.params.errors[0], message: e.message }
```

Comment:
Can we just ignore the rest of the errors?

Reply:
I tested a number of different scenarios and found that each `errorMessage` type only contains one error.  Testing included triggering both `additionalProperties` and `type`, with a custom error message for everything then for each type, and finally turning on the `singleError` option when initializing `ajv-errors`.